niworkflows/tests/conftest.py

```diff
@@ -34,7 +34,7 @@
 
 def _run_interface_mock(objekt, runtime):
     runtime.returncode = 0
-    runtime.endTime = dt.isoformat(dt.utcnow())
+    runtime.endTime = dt.isoformat(dt.now(dt.timezone.utc))
```

Comment:
I think you need to change L26 to:

```diff
-from datetime import datetime as dt
+import datetime as dt
```

And thiis would need to become:

```suggestion
    runtime.endTime = dt.datetime.isoformat(dt.datetime.now(dt.timezone.utc))
```

Reply:
Ouch! Indeed, sorry about that. I keep mixing up these two levels of `datetime.datetime`.